source/about-mongosync.txt

```diff
@@ -157,6 +157,31 @@ If the reverse sync is successful, ``mongosync`` enters the ``RUNNING`` state.
 The synchronization continues in the reverse direction from the original 
 sync job. 
 
+.. _mongosync-considerations:
+
+Considerations
```

Comment:
Why did we change this from considerations about continuous sync?

Reply:
hm looks like that's a mistake on my end — fixing.